src/services/jobs/jobs.class.js

```diff
@@ -74,19 +75,27 @@ class Service {
   }
 
   async patch(id, data, params) {
-    const where = {};
-    if (params.user.id) {
-      where.creatorId = params.user.id;
-    } else {
-      where['$creator.profile.uid$'] = params.user.uid;
-    }
-    return this.SequelizeService.patch(id, {
+    const where = {
+      creatorId: params.user.id,
+    };
+    debug(`[patch] id:${id}, params.user.uid:${params.user.uid}, where:`, where);
+    return this.sequelizeService.patch(id, {
       status: data.sanitized.status,
     }, { where });
   }
 
   async remove(id, params) {
-    return { id };
+    debug(`[remove] id:${id}, params.user.uid:${params.user.uid}`);
+    return this.sequelizeService.bulkRemove({
+      id,
+      creatorId: params.user.id,
+      status: ['RIP', 'DON'],
```

Comment:
Would be great to create constants for `RIP` and `DON` with explanation what it means.